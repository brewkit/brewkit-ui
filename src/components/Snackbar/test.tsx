import React from 'react';
import renderer from 'react-test-renderer';
import SnackbarProvider, { useSnackbar } from './index';


describe('Snackbar', () => {


    it('renders correctly', () => {
        const tree = renderer
            .create((
                <SnackbarProvider>
                    <div>foo</div>
                </SnackbarProvider>
            ))
            .toJSON();
        expect(tree).toMatchSnapshot();
    });


});
