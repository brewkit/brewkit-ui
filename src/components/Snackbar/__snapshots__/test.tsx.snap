// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snackbar renders correctly 1`] = `
Array [
  <div>
    foo
  </div>,
  <div />,
]
`;
